src/javascript/app/pages/form/currency-dropdown.jsx

```diff
@@ -26,7 +26,10 @@ const CurrencyContent = ({ currency_list, currency, onUpdate, elementId }) => {
     );
 };
 export const CurrencyDropdown = ({ currency_list, currency, onUpdate, elementId }) => (
-    <CustomDropdown value={currency}>
+    <CustomDropdown
+        value={currency}
+        fullHeightOnOpen={false}
```

Comment:
we better change the prop name to `fitContent`

so we don't can pass it like 

```
  <CustomDropdown
        value={currency}
        fitContent  
/>
```

Reply:
This is an existing props name we have. I think @Nuzhy-Deriv just use the props from actionsheet component